### ISSUE TITLE
How can I scroll the content in the preview window? Is it possible to scroll the content in the preview window?

### ISSUE THREAD

**Issue by bpan2020** [tablacus/TablacusExplorer#786]
How can I scroll the content in the preview window? Is it possible to scroll the content in the preview window?

**Comment by ccfs** [comment]
If "preview window" is the add-on:
![20241026_094437](https://github.com/user-attachments/assets/e441959d-aeca-4d2c-8031-ee56cb8d8487)

Mouse (method 1)
1. Click the Preview window
2. Scroll the wheel

Mouse (method 2)
1. Middle-click the text box in the Preview window
2. Move your mouse down and up

Keyboard
1. Click the text box in the Preview window
2. Press these keys:
	- PageDown
	- PageUp
	- Ctrl+End
	- Ctrl+Home
---
If "preview window" is the preview pane under Frame mode:
![20241026_094734](https://github.com/user-attachments/assets/57222487-7191-43dc-9c8c-fe986490738b)

Mouse (method 1)
1. Move your mouse pointer on the pane
2. Scroll the wheel

Mouse (method 2)
1. Middle-click the pane
2. Move your mouse down and up

Keyboard
1. Click the pane
2. Press these keys:
	- Down arrow
	- Up arrow
	- PageDown
	- PageUp
	- Ctrl+End
	- Ctrl+Home

**Comment by bpan2020** [comment]
Thanks. Looks I was using this mode "If "preview window" is "Preview" add-on". How can I switch to this mode:"If "preview window" is the preview pane under Frame mode:"

**Comment by ccfs** [comment]
Right-click the tab and choose "Show frames". Then click the toggle button if the preview pane does not show.
![fm3](https://github.com/user-attachments/assets/9199323f-5df5-48c5-8690-372ff236a747)

**Comment by bpan2020** [comment]
Thanks. I have another question. How to preview ".log" txt file?

**Comment by bpan2020** [comment]
Thanks. But how can I preview a '.log' text file in this preview window? It looks like 'show frames' is set per tab. Can we set it globally once and apply it to all tabs?

**Comment by ccfs** [comment]
**Supported file types**

This preview pane is a Windows Explorer feature. To add a supported file type, you can use the [Registry Editor](https://blog.ninethsense.com/technology/tips-tricks/enable-windows-explorer-previews-for-custom-file-types/) or a [File Explorer add-ons utility](https://learn.microsoft.com/en-us/windows/powertoys/file-explorer).

![reg](https://github.com/user-attachments/assets/78a88107-302e-462c-9f58-2761359a135b)

**Show frames**

To turn it on for all tabs:
1. Switch to a tab which has "Show frames" OFF.
2. TICK "Show frames" and "Default" on "Options > List" page and press the "OK" button.

![tab](https://github.com/user-attachments/assets/ecf96f32-bc72-495e-922c-7f98c9c3142f)
![sf](https://github.com/user-attachments/assets/42240913-af1b-45c8-8717-849456f83d02)

**Preview pane visibility**

When you starts TE, opens a new tab, or turns "Show frames" on, whether the pane is visible or not depends on last use. To have a better control of this status, you can use "Frame plus" add-on. It is not an official add-on but you can download it [here](https://github.com/tablacus/TablacusExplorer/issues/692).

**Comment by bpan2020** [comment]
Thanks. It seems that I still cannot preview ".log" files on one of my computers. For this computer, I switched to using the "preview window" add-on. However, the photo size displayed in this preview window is very small (the one on the left side). The one on the right side is displayed in the "preview" add-on window.
![图片](https://github.com/user-attachments/assets/2fe5b881-1e18-4e00-95b5-01c7f6f343dd)

**Comment by ccfs** [comment]
I do not quite understand. It seems that the file is NOT a photo but is a HTML file.

**Comment by bpan2020** [comment]
Sorry. It's a '.svg' file.

**Comment by ccfs** [comment]
Please zip and post the svg file for analysis.

**Comment by bpan2020** [comment]
Sure. Here it is. 
[demo.zip](https://github.com/user-attachments/files/17540641/demo.zip)

**Comment by ccfs** [comment]
"Preview Window" add-on has some problems about SVG:
- the graphics may not fully occupied available window space (your case)
- scroll bars may show up unnecessarily

I have tried changing the add-on scripts but not succcessful so far. So we have to live with it as is at the moment.

**Comment by bpan2020** [comment]
OK, thanks. I am currently using the 'Preview' add-on for SVG files and the 'Preview Window' add-on for other files. Should we file an issue for this?

**Comment by bpan2020** [comment]
Is there a quick way to open and close the preview window of the 'preview' add-on? For example, in the 'preview window' add-on, there is a button in the toolbar that allows this functionality.

**Comment by ccfs** [comment]
You can use "Add-on switcher" add-on to switch on/off another add-on. To toggle "Preview" add-on using a Tool bar button:
![20241031_163342](https://github.com/user-attachments/assets/eeda5baf-ea1e-4649-9b00-fe22ed0b3378)

**Comment by ccfs** [comment]
I have solved the small SVG preview problem. Please install this [patch](https://github.com/tablacus/TablacusExplorer/issues/692).
I do not have a solution to the scroll bar problem. So it will remain at the moment.

**Comment by bpan2020** [comment]
Great! Thanks!
